test/jasmine/tests/plot_api_test.js

```diff
@@ -1195,6 +1195,77 @@ describe('Test plot api', function() {
             .then(done);
         });
 
+        it('turns on cauto when cmid is edited', function(done) {
```

Comment:
this is an odd description... `cmid` only has effect if `cauto` is true, but doesn't turn on `cauto` per se, right?

Reply:
> but doesn't turn on `cauto` per se, right?

yes, it has to. If not, `restyle(gd, 'marker.cmid', 0)` wouldn't work.

Reply:
by "work" here I mean it wouldn't apply that new cmid value when `cmin` and `cmax` are set.

Reply:
OK I see what you mean.